plugin/open_url.vim

```diff
@@ -3,13 +3,13 @@ if exists('g:loaded_open_url')
 endif
 let g:loaded_open_url = 1
 
-function! s:SetGlobalOpt(opt, val)
-  if !exists('g:'.a:opt)
-    let g:{a:opt} = a:val
-  endif
-endfunction
+if !exists('g:open_url_browser_default')
+  let g:open_url_browser_default = 'xdg-open'
+endif
 
-call s:SetGlobalOpt('open_url_browser_default', 'sensible-browser')
```

Comment:
Why did you remove this ? There's no need to change the default offered by the plugin, if you want to override / change this config, set it in your `~/.vimrc`

Reply:
`xdg-open` (`xdg-utils`) is usually installed by default, while `sensible-browser` (`sensible-utils`) not. And even after installing this package, it would still not detect `firefox` and require me to set BROWSER environment variable.
And thus, I changed the default browser, as it should be something what most users already use.

Also, in README was no mention about `g:open_url_browser_default`, so nobody would know that this option exist without reading a code (what - let's be honest - most people don't do).